.env.example

```diff
@@ -8,8 +8,6 @@ COGNITE_BASE_URL=
 COGNITE_STATE_DB=
 COGNITE_STATE_TABLE=
 COGNITE_EXTRACTION_PIPELINE=
-DPS_TABLE_NAME=
```

Comment:
Why were these removed?  There still appear to be hard coded table values in the example_config, are they supposed to be hard coded?

Reply:
Yeah we need to introduce these back in that was probably done in the middle of moving stuff around in the config and reverting back to some of the old stuff. I will introduce these back in

Reply:
Done